src/utils.py

```diff
@@ -310,3 +312,47 @@ def get_response_data(json_response, data_default):
     Data default parameters takes what should data to be returned.
     """
     return json_response.get("result", {}).get("data", data_default)
+
+
+def server_run_flow(flow_name, flow_args):
+    """Run a flow.
```

Comment:
I hope we will have the ingestion API soon :)

Reply:
Yes, Michal. You are right